src/02-common-principles.md

```diff
@@ -449,7 +449,7 @@ using an object containing the following fields:
 | Description  | RECOMMENDED           | [string][]                | Description of the column.                                                                                      |
 | Levels       | RECOMMENDED           | [object][] of [strings][] | For categorical variables: An object of possible values (keys) and their descriptions (values).                 |
 | Units        | RECOMMENDED           | [string][]                | Measurement units. SI units in CMIXF formatting are RECOMMENDED (see [Units](./02-common-principles.md#units)). |
-| TermURL      | RECOMMENDED           | [string][]                | URL pointing to a formal definition of this type of data in an ontology available on the web.                   |
+| TermURL      | RECOMMENDED           | [string][]                | [URI](../02-common-principles.md#uniform-resource indicator) pointing to a formal definition of this type of data in an ontology available on the web.                   |
```

Comment:
To me (as a user) it feels awkward o have the keyname mention "URL" but definition asking for an URI.

Reply:
True, these fields should stay at URL, which is a specific subclass of URI. In the validator, we can still start to validate them as URI --> and it will not break datasets that have been previously valid.

Reply:
okidoki will revert any of the changes where the keyname contains URL.